WooCommerce/WooCommerce.xcodeproj/project.pbxproj

```diff
@@ -8066,7 +8070,7 @@
 				DEC51A9D274F8528009F3DF4 /* JetpackInstallStepsViewModel.swift in Sources */,
 				455DC3A327393C7E00D4644C /* OrderDatesFilterViewController.swift in Sources */,
 				45B6F4EF27592A4000C18782 /* ReviewsView.swift in Sources */,
-				028BAC4222F30B05008BB4AF /* StoreStatsV4PeriodViewController.swift in Sources */,
+				028BAC4222F30B05008BB4AF /* OldStoreStatsV4PeriodViewController.swift in Sources */,
```

Comment:
I missed this earlier - I'm not sure if we should include this change. Maybe discard it and merge `trunk` back to this branch instead?